taskt/Core/Automation/Commands/Data Commands/GetListItemCommand.cs

```diff
@@ -1,73 +1,76 @@
-﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
+﻿using Microsoft.Office.Interop.Outlook;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using System.Collections.Generic;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
-using Microsoft.Office.Interop.Outlook;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to get an item from a List")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to get an item from a List.")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Data Commands")]
+    [Description("This command returns an item (having a specific index) from a list.")]
     public class GetListItemCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the List Name")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a existing List.")]
-        [Attributes.PropertyAttributes.SampleUsage("**myData**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("List")]
+        [InputSpecification("Provide a list variable.")]
+        [SampleUsage("{vList}")]
+        [Remarks("Any type of variable other than list will cause error.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_ListName { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please enter the index of the List item")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a valid List index value")]
-        [Attributes.PropertyAttributes.SampleUsage("0 or [vIndex]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Index")]
+        [InputSpecification("Specify a valid list item index.")]
+        [SampleUsage("0 || {vIndex}")]
+        [Remarks("'0' is the index of first item in a list. Invalid or out of bound index will cause an error.")]
```

Comment:
Change to "'0' is the index of the first item in a list. Providing an invalid or out-of-bounds index will result in an error."

Reply:
done